src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/Main.cs

```diff
@@ -184,5 +207,44 @@ public string GetTranslatedPluginTitle()
         {
             return Properties.Resources.Microsoft_plugin_sys_plugin_name;
         }
+
+        private bool ExecuteCommand(string confirmationMessage, Action command)
+        {
+            if (this._context.CurrentPluginMetadata.ConfirmSystemCommands)
+            {
+                MessageBoxResult messageBoxResult = MessageBox.Show(
+                    confirmationMessage,
+                    Properties.Resources.Microsoft_plugin_sys_confirmation,
+                    MessageBoxButton.YesNo,
+                    MessageBoxImage.Warning);
+
+                if (messageBoxResult == MessageBoxResult.No)
+                {
+                    return false;
+                }
+            }
+
+            command();
+            return true;
+        }
+
+        public Control CreateSettingPanel()
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
Why do we need this method? Remove it.

Reply:
If I remove this, I get 'Main' does not implement interface member ISettingProvider.CreateSettingsPanel()?

Reply:
It's not used but anyway let's leave it there.